lib/players/human.rb

```diff
@@ -0,0 +1,10 @@
+module Players
+
+class Human < Player
+	def move(input)
+		puts "Where would you like to move?"
+		input = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```